packages/backend/src/payment-method/ilp/connector/core/middleware/balance.ts

```diff
@@ -20,6 +20,12 @@ export function createBalanceMiddleware(): ILPMiddleware {
     next: () => Promise<void>
   ): Promise<void> => {
     const { amount } = request.prepare
+    const logger = services.logger.child(
+      { module: 'balance-middleware' },
+      {
+        redact: ['transferOptions.destinationAccount.http.outgoing.authToken']
```

Comment:
I would say yes but the wildcard `*token` doesnt work like that. https://github.com/davidmarkclements/fast-redact?tab=readme-ov-file#wildcards

It doesn't match `token`, `authToken`, etc. [nor would it look for tokens at all depths](https://github.com/davidmarkclements/fast-redact/issues/5).

From what I can tell there isn't a way to configure redact to redact all values for a key matching `*token` at any depth, nor even a key matching `*token` at a specific depth.

Reply:
Makes sense, I'm guessing recursive redacting is quite costly. In that case, maybe we just do what you had previously, or instead of deleting keys how you had before, we can just selectively log a few properties of this object. Sorry for the back-and-forth, thought this was recursive from the beginning

Reply:
As we talked off github, it is probably easier to blacklist sensitive info than whitelist useful info here because there is a fair amount and not exactly sure what will matter. So it currently redacts just the `authToken`.